### PR TITLE
create animaton

### PR DESCRIPTION
**Pull request by RobertDebicki**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
Develop an interactive, moving news ticker according to provided assets.

### Requirements
1. Text should be moving with speed enabling reading
2. Animation should be set to linear
3. While hovering the animation should stop
4. Animation should be looped indefinitely

### Code Changes Review

#### challenge-2/index.html
The changes in the HTML file seem to be in line with the requirements. The text is now divided into two paragraphs, each with a different animation class. This could be a way to achieve the indefinite loop of the animation.

#### challenge-2/style.css
The CSS changes also seem to meet most of the requirements. The animations are set to linear and looped indefinitely. The speed of the animation is set to 10000ms, which should allow for the text to be read.

However, there is a missing requirement in the CSS changes:
- There is no hover effect that stops the animation. This can be achieved by adding a hover state to the `.txt` class that sets the animation-play-state property to 'paused'.

Here is an example of how this could be implemented:

```css
.txt:hover {
    animation-play-state: paused;
}
```

In conclusion, the code changes cover most of the task requirements, but there is one missing requirement: stopping the animation on hover.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
# Code Review Report

## Task Goal
Develop an interactive, moving news ticker according to provided assets.

## Requirements
1. Text should be moving with speed enabling reading
2. Animation should be set to linear
3. While hovering the animation should stop
4. Animation should be looped indefinitely

## Code Changes Review

### challenge-2/index.html
The changes in the HTML file seem to be in line with the task requirements. The text has been split into two paragraphs, each with a different animation class. This will allow for the looping animation of the text.

### challenge-2/style.css
The changes in the CSS file also align with the task requirements. The animations `slide-left` and `slide-left2` have been defined and applied to the text paragraphs in the HTML file. The animations are linear and loop indefinitely, which meets requirements 2 and 4.

However, there seems to be a missing requirement in the CSS changes. The requirement that the animation should stop while hovering is not implemented in the provided code changes. This can be achieved by adding a `:hover` pseudo-class to stop the animation.

Here is a suggestion on how to implement this:

```css
.animation1:hover, .animation2:hover {
    animation-play-state: paused;
}
```

This will pause the animation when the mouse hovers over the text, fulfilling requirement 3.

In conclusion, the code changes cover most of the task requirements except for the hover functionality to pause the animation.
